rpcs3/rpcs3qt/game_compatibility.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <memory>
+#include <QtGui/QPainter>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QtNetwork/QNetworkAccessManager>
+#include <QtNetwork/QNetworkReply>
+#include <QtNetwork/QNetworkRequest>
+#include <QTableWidget>
+
+#include "gui_settings.h"
+
+class game_compatibility
+{
+private:
+	int m_compatibilityColumn;
+	int m_serialColumn;
+	std::shared_ptr<gui_settings> xgui_settings;
+	std::unique_ptr<QNetworkAccessManager> m_networkAccessManager;
+	QTableWidget *m_tableWidget;
+
+public:
+	game_compatibility(int compatibilityColumn, int serialColumn, std::shared_ptr<gui_settings> settings, QTableWidget *tableWidget);
+	bool isEnabled();
+	void update();
+};
+
+class game_compatibility_pixmap : public QPixmap
+{
+private:
+	std::unique_ptr<QPainter> m_painter;
+
+public:
+	game_compatibility_pixmap(QColor color) : QPixmap(16, 16)
+	{
+		fill(Qt::transparent);
+
+		m_painter = std::make_unique<QPainter>(this);
```

Comment:
Does it really need a painter as a member? It's used only once here. Could be just a local var.

Reply:
Not really indeed. Use a local var now, thanks.